src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ExplicitConversionSymbolReferenceFinder.cs

```diff
@@ -43,20 +46,18 @@ protected override bool CanFind(IMethodSymbol symbol)
 
         var underlyingNamedType = GetUnderlyingNamedType(symbol.ReturnType);
         Contract.ThrowIfNull(underlyingNamedType);
-        var documentsWithName = await FindDocumentsAsync(project, documents, cancellationToken, underlyingNamedType.Name).ConfigureAwait(false);
-        var documentsWithType = await FindDocumentsAsync(project, documents, underlyingNamedType.SpecialType.ToPredefinedType(), cancellationToken).ConfigureAwait(false);
 
         using var _ = ArrayBuilder<Document>.GetInstance(out var result);
```

Comment:
nit (and you're preserving logic, so up to you), but chaneg this to a pooled hashset.  it was an array before to keep all the sigs the same.  but now that this is callbac style, this can change.